disnake/abc.py

```diff
@@ -96,7 +95,7 @@
     from .ui.view import View
     from .user import ClientUser
 
-    MessageableChannel = Union[TextChannel, Thread, DMChannel, PartialMessageable, VoiceChannel]
+    MessageableChannel = Union[GuildMessageable, DMChannel, PartialMessageable]
```

Comment:
why is this defined here when multiple files import it from message.py?

Reply:
Good point, changed the imports in 091ca9af875cdc961e74fcd2ee5af2c79a140ccc